src/ng/directive/ngAs.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name ngAs
+ * @restrict A
+ *
+ * @description
+ * The `ngAs` attribute tells Angular to assign element component controller
+ * to a given property.
+ *
+ * Using this directive you can use the controller of existing components
+ * in your template (children components).
+ *
+ * If the children component is destroyed
```

Comment:
Not really sure what our conventions are here, but this line looks pretty short...?